docs/modules/reference/pages/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -273,43 +280,60 @@ Note that you must include the `monitor` section for each service in your defini
-      <page scheme="https"
-            method="POST"
-            host="ecomm.example.com" port="443"
-            path="/ecomm/controller"
-            virtual-host="ecomm.example.com"
-            successMatch="requesttab_select.gif"
-            failureMessage="Login failed: ${1}"
-            timeout="10000"
-            http-version="1.1">
-        <parameter key="action_name" value="XbtnLogin"/>
-        <parameter key="session_timeout" value=""/>
-        <parameter key="userid" value="EXAMPLE"/>
-        <parameter key="password" value="econ"/>
+      <page scheme="http" <1>
+            host="ecomm.example.com" <2>
+            port="80" <3>
+            path="/ecomm/jsp/Login.jsp" <4>
+            virtual-host="ecomm.example.com" <5>
+            successMatch="eComm Login" <6>
+            timeout="10000" <7>
+            http-version="1.1"/> <8>
+      <page scheme="https" <1>
+            method="POST" <9>
+            host="ecomm.example.com" port="443" <2>
+            path="/ecomm/controller" <4>
+            virtual-host="ecomm.example.com" <5>
+            successMatch="requesttab_select.gif" <6>
+            failureMessage="Login failed: ${1}" <10>
+            timeout="10000" <7>
+            http-version="1.1"> <8>
+        <parameter key="action_name"
+                   value="XbtnLogin"/>
+        <parameter key="session_timeout"
+                   value=""/>
+        <parameter key="userid"
+                   value="EXAMPLE"/>
+        <parameter key="password"
+                   value="econ"/>
       </page>
-      <page scheme="http"
-            host="ecomm.example.com" port="80"
-            path="/econsult/controller"
-            virtual-host="ecomm.example.com"
-            successMatch="You have successfully logged out of eComm"
-            timeout="10000" http-version="1.1">
-        <parameter key="action_name" value="XbtnLogout"/>
+      <page scheme="http" <1>
+            host="ecomm.example.com" <2>
+            port="80" <3>
+            path="/econsult/controller" <4>
+            virtual-host="ecomm.example.com" <5>
+            successMatch="You have successfully logged out of eComm" <6>
+            timeout="10000" <7>
+            http-version="1.1"> <8>
+        <parameter key="action_name"
+                   value="XbtnLogout"/>
       </page>
     </page-sequence>
   </parameter>
 </service>
 
-<monitor service="OpenNMS-Web-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <1>
+<monitor service="OpenNMS-Web-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <11>
 ----
-<1> Required *monitor* section.
+<1> Define the URL scheme as http or https.
+<2> Set host field in HTTP header.
+<3> Host port number to use when connecting.
+<4> The relative URL to call in the request.
+<5> Set the virtual host field in the HTTP header.
+<6> Text to look for in the response body.
+<7> Timeout for the isReachable method, in milliseconds.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.